### ISSUE TITLE
Can't install RealmSwift 5.5.2 with Cocoapods

### ISSUE THREAD

**Issue by bb-git** [realm/realm-cocoa#7263]
I tried everything, also `pod deintegrate`. RealmSwift 5.5.1 installs without any issues.

## Goals
Install the latest version of Realm 5.5.2

## Actual Results
```
[!] CocoaPods could not find compatible versions for pod "RealmSwift":
  In Podfile:
    RealmSwift (= 5.5.2)

None of your spec sources contain a spec satisfying the dependency: `RealmSwift (= 5.5.2)`.

You have either:
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
```

## Steps for others to Reproduce
pod install --repo-update

## Version of Realm and Tooling

Realm framework version: ?
5.5.1
Xcode version: ?
12.5
iOS/OSX version: ?
11.3.1
Dependency manager + version: ?
cocoapods 1.10.1

**Comment by leemaguire** [comment]
Hi @bb-git We are looking into this now.

**Comment by tgoyne** [comment]
5.5.2 is just a republish of 5.5.1 to add Xcode 12.5 binaries, which isn't applicable to cocoapods.

**Comment by bb-git** [comment]
Thanks for letting me know, I was hoping it will fix the crashes from 5.5.1. Will consider upgrading to 10.7.6 then.
